### ISSUE TITLE
Use encryption at rest on bucket as default.

### ISSUE THREAD

**Issue by jayzalowitz** [cloudposse/terraform-aws-ecs-codepipeline#48]
In order for this to be in some envs consider adding AES-256 encryption by default to your buckets.

**Comment by nitrocode** [comment]
Yes that's a good idea as that would be more secure. We welcome pull requests. Please become a contributor if you feel strongly about this.
